src/module/updateTask.js

```diff
@@ -6,4 +6,11 @@ const updateTask = (index, description) => {
     tasks[index].description = description;
   }
 };
-export default updateTask;
\ No newline at end of file
+
+const completeTask = (index, completed) => {
+  index = tasks.findIndex((object) => object.index === index);
+  if (index !== -1) {
+    tasks[index].completed = completed;
+  }
+};
+export { updateTask, completeTask };
```

Comment:
- Hey, great job using the findIndex method here, however, according to the project requirement, you should use the `filter` method. please, kindly refactor to meet the project requirement below:

![Screenshot from 2023-01-19 17-56-42](https://user-images.githubusercontent.com/59397159/213508807-8b79f2bc-ddde-4a69-a46b-788f744d7767.png)

Reply:
@zieeco thanks changed find index method to filter as per your recommendation.